slog/example_test.go

```diff
@@ -0,0 +1,68 @@
+package slog
+
+import (
+	"context"
+	"errors"
+
+	"go.opencensus.io/trace"
+)
+
+func ExampleNewContext() {
```

Comment:
I'd combine `ExampleNewContext` and `ExampleFromContext` to show the full end to end.